lib/internal/vm/module.js

```diff
@@ -76,6 +78,13 @@ const kLink = Symbol('kLink');
 
 const { isContext } = require('internal/vm');
 
+function isModule(object) {
+  if (typeof object !== 'object' || object === null || !ObjectPrototypeHasOwnProperty(object, kWrap)) {
```

Comment:
This might be a little bit better for handling `null` and `undefined` values.

```suggestion
  if (typeof object !== 'object' || object == null || !ObjectPrototypeHasOwnProperty(object, kWrap)) {
```

Reply:
This tries to be explicit since an `undefined` value has been guarded by `typeof object !== 'object'`.